rpcs3/Emu/RSX/CgBinaryProgram.h

```diff
@@ -365,7 +365,7 @@ class CgBinaryDisasm
 
 				m_arb_shader += fmt::format("#%d ", i) + param_type + param_name + param_semantic + param_const + "\n";
 
-				offset += sizeof(CgBinaryParameter);
+				offset += static_cast<CgBinaryParameterOffset>(sizeof(CgBinaryParameter));
```

Comment:
u32 is the underlying type of it. (below be_t<u32>);
```suggestion
				offset += u32{sizeof(CgBinaryParameter)};
```